stable/grafana/values.yaml

```diff
@@ -143,3 +143,35 @@ grafana.ini:
     mode: console
   grafana_net:
     url: https://grafana.net
+
+## Grafana's LDAP configuration
+## Templated by the template in _helpers.tpl
+## NOTE: To enable the grafana.ini must be configured with auth.ldap.enabled
+## ref: http://docs.grafana.org/installation/configuration/#auth-ldap
+## ref: http://docs.grafana.org/installation/ldap/#configuration
+ldap.toml: {}
```

Comment:
Can you please document the empty default in the readme

Reply:
Added

Reply:
I think is better to use `ldap` instead of `ldap.toml` as key, if not is always problematic to reference keys with dot in it, for example when using `--set`